packages/loot-core/src/shared/schedules.js

```diff
@@ -42,39 +42,47 @@ export function getHasTransactionsQuery(schedules) {
     .select(['schedule', 'date']);
 }
 
-function makeNumberSuffix(num) {
-  // Slight abuse of date-fns to turn a number like "1" into the full
-  // form "1st" but formatting a date with that number
-  return monthUtils.format(new Date(2020, 0, num, 12), 'do');
-}
-
-function prettyDayName(day) {
+function prettyDayName(day, locale) {
   let days = {
-    SU: 'Sunday',
-    MO: 'Monday',
-    TU: 'Tuesday',
-    WE: 'Wednesday',
-    TH: 'Thursday',
-    FR: 'Friday',
-    SA: 'Saturday'
+    SU: new Date('2020-01-06T00:00:00.000Z'),
+    MO: new Date('2020-01-07T00:00:00.000Z'),
+    TU: new Date('2020-01-08T00:00:00.000Z'),
+    WE: new Date('2020-01-09T00:00:00.000Z'),
+    TH: new Date('2020-01-10T00:00:00.000Z'),
+    FR: new Date('2020-01-11T00:00:00.000Z'),
+    SA: new Date('2020-01-12T00:00:00.000Z')
```

Comment:
This can be improved. I think that the name in english as "key" will be better option.

Reply:
Unfortunately I’m pretty sure those values are the names stored in the database for existing schedules, so they can’t be changed.